less/ovh-angular-input-number-spinner.less

```diff
@@ -11,6 +11,10 @@
     }
 
     .input-group-btn:first-child > button.btn {
```

Comment:
What's about writing nested rules like this:

```less
.input-group-btn:first-child {
    & > button.btn {
        margin-right: 0px;
    }

    & + input.form-control {
        padding: 0;
    }
}
```

Reply:
Yup.